MechJeb2/MechJebModuleNodeExecutor.cs

```diff
@@ -124,11 +132,22 @@ public override void OnFixedUpdate()
 
             double timeToNode = node.UT - vesselState.time;
             //(!double.IsInfinity(num) && num > 0.0 && num2 < num) || num2 <= 0.0
-            if ((!double.IsInfinity(halfBurnTime) && halfBurnTime > 0 && timeToNode < halfBurnTime) || timeToNode < 0)
-            {
-                burnTriggered = true;
-                if (!MuUtils.PhysicsRunning()) core.warp.MinimumWarp();
-            }
+			if (mode == Mode.ONE_NODE)
+			{
+				if ((!double.IsInfinity(halfBurnTime) && halfBurnTime > 0 && timeToNode < halfBurnTime) || timeToNode < 0)
+				{
+					burnTriggered = true;
+					if (!MuUtils.PhysicsRunning()) core.warp.MinimumWarp();
+				}
+			}
+			else if (mode == Mode.ONE_PNODE)
+			{
+				if ((!double.IsInfinity(halfBurnTime) && halfBurnTime > 0 && timeToNode <= 0.0225) || timeToNode < 0)
+				{
+					burnTriggered = true;
+					if (!MuUtils.PhysicsRunning()) core.warp.MinimumWarp();
+				}
+			}
```

Comment:
can you reset your editor to keep tabs as spaces and not convert?  i wonder if visual studio respects some kind of magic comment policy...